src/pages/media-kit/index.page.tsx

```diff
@@ -0,0 +1,233 @@
+import type { GetStaticProps, NextPage } from 'next'
+import Head from 'next/head'
+import { useTranslation } from 'next-i18next'
+import { serverSideTranslations } from 'next-i18next/serverSideTranslations'
+import { Page } from '../../components/Page'
+import { StyledLink } from '../../components/StyledLink'
+import EmbellishedLeft from './embellished/left.svg'
+import EmbellishedRight from './embellished/right.svg'
+import styles from './index.module.scss'
+
+const MediaKit: NextPage = () => {
+  const { t } = useTranslation(['media-kit', 'common'])
+
+  const sections: Array<{
+    title: string
+    description: string
+    download?: Record<'label' | 'link', string>
+    link?: Record<'label' | 'link', string>
+    assets: React.ReactElement
+  }> = [
+    {
+      title: t('logo.title'),
+      description: t('logo.description'),
+      download: {
+        label: t('logo.download'),
+        link: '/assets/media-kit/Nervos Logo.zip',
+      },
+      assets: (
+        <div>
+          {['light', 'dark'].map(type => (
+            <img
+              key={type}
+              alt={`${type}-logo`}
+              src={`/images/media-kit/logo/${type}.png`}
+              loading="lazy"
+              width="300"
+              height="300"
+            />
+          ))}
+        </div>
+      ),
+    },
+    {
+      title: t('symbol.title'),
+      description: t('symbol.description'),
+      download: {
+        label: t('symbol.download'),
+        link: '/assets/media-kit/Nervos Token Assets.zip',
+      },
+      assets: (
+        <div>
+          {['light', 'dark'].map(type => (
+            <img
+              key={type}
+              alt={`${type}-symbol`}
+              src={`/images/media-kit/symbol/${type}.png`}
+              loading="lazy"
+              width="300"
+              height="300"
+            />
+          ))}
+        </div>
+      ),
+    },
+    {
+      title: t('wordmark.title'),
+      description: t('wordmark.description'),
+      download: {
+        label: t('wordmark.download'),
+        link: '/assets/media-kit/Nervos Wordmark.zip',
+      },
+      assets: (
+        <div>
+          {['light', 'dark'].map(type => (
+            <img
+              key={type}
+              alt={`${type}-wordmark`}
+              src={`/images/media-kit/wordmark/${type}.png`}
```

Comment:
I think these resources are better off being placed directly in the `src\pages\media-kit` directory, as they are exclusive to it and there is no need to elevate them to shared resources across multiple components.

This can achieve the goal mentioned in https://github.com/Magickbase/ckb-explorer-public-issues/issues/367#issuecomment-1704867805 of `Make the implementation of each component more cohesive`

Reply:
How can I access to them via url when they are placed outside the `public` directory

Reply:
https://github.com/Magickbase/nervos-official-website/assets/7271329/32452f77-be6e-47ba-8fb5-5a1055937bfc

Encountered a weird bug of Image 🤔

Reply:
> Encountered a weird bug of Image 🤔

If the original width of the image is 300, then the width and height attributes should be removed, because the `StaticImageData` obtained from import img has this attribute and it will be used by `next/Image`.

If the original image is not 300 wide, consider using `max-width: 300px; height: auto;`

Reply:
It's not caused by size but by image optimization of next/image
![image](https://github.com/Magickbase/nervos-official-website/assets/7271329/6dc51fb7-b703-4110-8ed6-151c594dc105)

Reply:
> It's not caused by size but by image optimization of next/image

Has this issue been resolved? Can resources that belong only to the media-kit page be moved to the implementation folder of its component?